drivers/rs485_linux/work/arbiter/implem/default/CPP/src/arbiter_state.h

```diff
@@ -0,0 +1,37 @@
+#include "dataview-uniq.h"
+
+#include <atomic>
+#include <chrono>
+
+#define QUEUE_SIZE 1000
```

Comment:
This might be quite short, and on Linux we have all the memory we need. Let's make it 65536

Reply:
done